src/MakersOfDenmark.Domain/Models/MakerSpace.cs

```diff
@@ -16,6 +18,15 @@ public class MakerSpace : Entity<Guid>
         public AccessType AccessType { get; set; }
         public Organization Organization { get; set; }
         public ICollection<Tool> Tools { get; set; } = new List<Tool>();
-
+        public ICollection<MODUserFollowMakerSpace> Followers { get; set; } = new List<MODUserFollowMakerSpace>();
+        public void FollowMakerSpace(MODUser newFollower)
+        {
+            Followers.Add(new MODUserFollowMakerSpace { User = newFollower, MakerSpace = this, CreatedAt = DateTimeOffset.UtcNow });
+        }
+        public void UnfollowMakerSpace(MODUser exFollower)
+        {
+            var isFollowed = Followers.FirstOrDefault(x => x.User == exFollower && x.MakerSpace == this);
```

Comment:
Vil der nogensinde være en situation hvor `&& x.MakerSpace == this); vil være true, andet end hvis referencen er ens?

Derudover så vil jeg mene, at de Followers du har på et MakerSpace aldrig vil være andet end MakerSpace's egne followers. Eller vil det?

Vi kunne overskrive equals metoden, således at de sammenlignes på Id når man bruger == operatoren. Dog vil jeg mene, at det blot at bedre at lave en FirstOrDefault(x => x.User.Id == exFollower.Id);

Reply:
Hvad hvis exFollower er null?

Reply:
ændret :)